spec/decorator_spec.rb

```diff
@@ -0,0 +1,23 @@
+require_relative '../school_library/decorators/capitalize_decorator'
+require_relative '../school_library/decorators/trimmer_decorator'
+require_relative '../school_library/library/person'
+
+describe CapitalizeDecorator do
+  describe '#should capitalize the name' do
+    it 'should capitalize the name' do
+      person = Person.new(1, name: 'maximilianus', age: 25)
+      capitalized = CapitalizeDecorator.new(person)
+      expect(capitalized.correct_name).to eql 'Maximilianus'
+    end
+  end
+end
+
+describe TrimmerDecorator do
+  describe '#should trim the name' do
+    it 'should trim the name' do
+      person = Person.new(1, name: 'maximilianus ', age: 25)
+      trimmed = TrimmerDecorator.new(person)
+      expect(trimmed.correct_name).to eql 'maximilianus'
+    end
+  end
```

Comment:
- [ ] This is a close one! but the `TrimmerDecorator` should return the first 10 characters so for the string `maximilianus` it should return `maximilian` so please fix this.